layouts/partials/form.html

```diff
@@ -69,4 +69,25 @@
   $(document).ready(function() {
     $('select').material_select();
   });
+
+  function connect() {
```

Comment:
So you mean instead of extracting the fields one by one, I just serialize the form input values? Okay! Thank you @Skaty